packages/v4/src/content/design-guidelines/usage-and-behavior/filters/filters.md

```diff
@@ -12,210 +12,315 @@ related: [
 
 #### When to use
-Use a faceted (checkbox) filter when:
-* The user is browsing a set of items without robust knowledge of what is in the set.
-* The important characteristics of the items to be filtered are mostly traits with a small and discrete set of choices like color, status, availability, or distributer rather than arbitrary text or numeric values such as name, date created, or amount of available space.
-* If the important characteristics of the items are mostly open-ended values, consider using an attribute-value filter instead.
+Use a filter group when you only have a handful of attributes that you want to show. Only use this filter when space allows; otherwise consider using an attribute-value or faceted filter instead.
+
+#### Behavior
+Filter queries are applied when the user selects or deselects an item in the list or selects a date/time range. If the filter group is only made up of single selects, the selected filters will be shown inside each filter’s toggle. If the filter group contains multiple selects, the selected filters will be shown as chips underneath the filters. Additionally, you may also choose to display the selected number of items in each filter as a badge in their toggle. A clear filter action to the right of the filters will remove all applied filters.
+
+#### Supported and features
+Filter groups support [filter chips](#filter-chip), [badges](#badge-count), and [type-ahead](#type-ahead). 
+
+**Filter chips** are recommended for filter groups that have one or more checkbox selects. If the filter group only consists of single selects, there is no need to add filter chips as the count will always be one.
+
+**Badges** can be used in a checkbox select toggle to indicate how many selections have been made for that attribute filter.
+
+**Type-ahead** allows the user to narrow down their search and is supported for all value selection types except checkbox selects. Although typeahead is **not supported** for checkbox selects, you can add a search input inside the menu, above the items the user can select from.
+
+### Toggle group
+A toggle group filter allows users to select from a set of predefined different elements to filter something on. This filter uses the [toggle group](/components/toggle-group) component — read more in the [toggle group guidelines](/toggle-group/design-guidelines).
+
+<img src="./img/toggle-group.png" alt="toggle group" width="541" />
+
+#### When to use
+Use a toggle group filter when you have few filter options that you want users to filter from, without hiding the options in a select list. A common usage for toggle groups is for filtering a chart view, to include or remove a data point from view.
```

Comment:
I would argue that's what we say already -- "when you have few filter options"? but I can re-word? What do you think

Reply:
I don't feel strongly about this one. When I read it I felt like it was implying usage in a chart/card, but maybe that's just me. If you can think of a better way to say this fine, otherwise I think it's OK.